flink-catalog-examples/README.md

```diff
@@ -0,0 +1,273 @@
+          ▓███▒ ███   ░▓▓▒░░   ░▓████▓░                  ░▒▓▒  █▓
+          █▓▒▒▓▓██  ░▒▒░░░▒▒▒▒▓██▓░                            █▓
+          ██ ▓░▒█   ▓▓▓▓▒░░  ▒█▓       ▒▓▓██▓    ▓▒          ▒▒▓
+          ▓█▓ ▓▒█  █▓░  ░▒▓▓██▒            ░▓█▒   ▒▒▒░▒▒▓█████▒
+           ██░ ▓█▒█▒  ▒▓▓▒  ▓█                █░      ░░░░   ░█▒
+           ▓█   ▒█▓   ░     █░                ▒█              █▓
+            █▓   ██         █░                 ▓▓        ▒█▓▓▓▒█░
+             █▓ ░▓██░       ▓▒                  ▓█▓▒░░░▒▓█░    ▒█
+              ██   ▓█▓░      ▒                    ░▒█▒██▒      ▓▓
+               ▓█▒   ▒█▓▒░                         ▒▒ █▒█▓▒▒░░▒██
+                ░██▒    ▒▓▓▒                     ▓██▓▒█▒ ░▓▓▓▓▒█▓
+                  ░▓██▒                          ▓░  ▒█▓█  ░░▒▒▒
+                      ▒▓▓▓▓▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░▓▓  ▓░▒█░
+
+    ______ _ _       _       _____  ____  _         _____ _ _            _  BETA   
+   |  ____| (_)     | |     / ____|/ __ \| |       / ____| (_)          | |  
+   | |__  | |_ _ __ | | __ | (___ | |  | | |      | |    | |_  ___ _ __ | |_
+   |  __| | | | '_ \| |/ /  \___ \| |  | | |      | |    | | |/ _ \ '_ \| __|
+   | |    | | | | | |   <   ____) | |__| | |____  | |____| | |  __/ | | | |_
+   |_|    |_|_|_| |_|_|\_\ |_____/ \___\_\______|  \_____|_|_|\___|_| |_|\__|
+
+        Welcome! Enter 'HELP;' to list all available commands. 'QUIT;' to exit.
+
+
+Flink SQL>
+```
+
+After that, we need to create a Pravega Catalog to access the data stored in Pravega.
+In order to use Pravega Catalog, we need to add schema registry group and register the schema of your serialization format to schema registry service.
+The datagen container has done these things, so the catalog can get the schema information, translate the Pravega stream into Flink table and show in the catalog.
+You can check this by calling Pravega Schema Registry's REST API:
+```
+docker-compose exec schemaregistry curl -X GET http://localhost:9092/v1/groups/userBehavior/schemas?namespace=examples
+```
+
+Then the datagen container will send data to `userBehavior` stream in `examples` scope of Pravega. 
+Pravega will send data to Flink as a data source by Pravega Flink connector. 
+
+We run the following DDL statement in SQL CLI to create a Flink catalog that connects to Pravega and schema-registry.
+We will use Avro as the format for serialization in table sink as predefined in datagen container.
+
+```sql
+CREATE CATALOG pravega WITH(
+    'type' = 'pravega',                                     --Flink catalog type
+    'controller-uri' = 'tcp://pravega:9090',                --Pravega controller URI
+    'schema-registry-uri' = 'http://schemaregistry:9092',   --Schema-Registry service URI
+    'default-database' = 'examples',                        --Flink catalog database, mapped to Pravega scope
+    'serialization.format' = 'Avro'                         --data serialization format
```

Comment:
Do NOT put meaningless comment, try to explain more like "it is the serialization format to be used against all the Pravega streams in the catalog", otherwise remove it.